### PR TITLE
Support for heketi 4 Release changes

### PR DESCRIPTION
**Pull request by MohamedAshiqrh**
Closes: #171 
Closes: #172 
Signed-off-by: Mohamed Ashiq Liyazudeen <mliyazud@redhat.com>

---
This change is [Reviewable](https://reviewable.io/reviews/gluster/gluster-kubernetes/173)

### CODE REVIEW

**Comment by MohamedAshiqrh** [comment]
@jarrpa @obnoxxx @humblec I have tested all the bits with Openshift and It works as expected. Please give it a try in kubectl too. I do not have a setup. kubectl is tested by @lpabon in heketi and @jarrpa for reviews, Just the changes in yaml file to be tested.

**Comment by MohamedAshiqrh** [comment]
@raghavendra-talur and @ramkrsna Please review.

**Comment by MohamedAshiqrh** [comment]
Also found issue #172 while testing.

**Comment by MohamedAshiqrh** [comment]
Fixed #172 in the same PR.

File: deploy/kube-templates/heketi-deployment.yaml [hunk 1]
```diff
@@ -1,4 +1,15 @@
 ---
+kind: Secret
+apiVersion: v1
+metadata:
+  name: heketi-db-backup
+  labels:
+    glusterfs: heketi-db
+    heketi: db
```
Comment (jarrpa) [inline deploy/kube-templates/heketi-deployment.yaml]: Do we need this label for anything?

File: deploy/ocp-templates/heketi-template.yaml [hunk 2]
```diff
@@ -11,6 +11,16 @@ metadata:
 labels:
   template: heketi
 objects:
+- kind: Secret
+  apiVersion: v1
+  metadata:
+    name: heketi-db-backup
+    labels:
+      glusterfs: heketi-db
+      heketi: db
```
Comment (jarrpa) [inline deploy/ocp-templates/heketi-template.yaml]: Same as above, do we need it for anything?

File: deploy/kube-templates/heketi-deployment.yaml (hunk 1 above)
Comment (MohamedAshiqrh) [inline deploy/kube-templates/heketi-deployment.yaml]: Could be helpful to search nothing in particular. Can remove not anything important, Do you want me to remove?

File: deploy/ocp-templates/heketi-template.yaml (hunk 2 above)
Comment (MohamedAshiqrh) [inline deploy/ocp-templates/heketi-template.yaml]: Could be helpful to search nothing in particular. Can remove not anything important, Do you want me to remove?

Comment (MohamedAshiqrh) [inline deploy/ocp-templates/heketi-template.yaml]: Addressed your comments just left one label for searching.

File: deploy/kube-templates/heketi-deployment.yaml (hunk 1 above)
Comment (MohamedAshiqrh) [inline deploy/kube-templates/heketi-deployment.yaml]: Addressed your comments just left one label for searching.

**Comment by ramkrsna** [comment]
LGTM .

**Pull request merged**